src/Samples/CSharp/Analyzers/CSharpAnalyzers/CSharpAnalyzers.Test/CSharpAnalyzers.Test.csproj

```diff
@@ -14,7 +14,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>CSharpAnalyzers.Test</RootNamespace>
     <AssemblyName>CSharpAnalyzers.Test</AssemblyName>
-    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
```

Comment:
You shouldn't make this change - this isn't a portable library.  It's already targeting desktop 4.6.